src/ReturnTypes/EloquentBuilderExtension.php

```diff
@@ -27,6 +27,13 @@ final class EloquentBuilderExtension implements DynamicMethodReturnTypeExtension
 {
     use Concerns\HasBroker;
 
+    private const NON_DYNAMIC_WHERES = [
```

Comment:
note: I may be able to figure this out via `hasNativeMethod`,  I'll look into it...